index.html

```diff
@@ -1,29 +1,319 @@
+
+      <br /><br />
+
+      <div class="row">
+        <div class="col-md-2"></div>
+        <div class="col-md-8">
+          <p>
+            <strong>BOSH</strong> is an incredibly powerful Open Source tool that offers a <strong>complete approach</strong> towards operating cloud software. You can use BOSH to deploy and orchestrate any piece of infrastructure. BOSH takes care of lifecycle management on many different levels, as well as automated health monitoring. The main reason BOSH succeeds in doing all this, lies in its <strong>deterministic approach</strong>. BOSH expects every single aspect of your infrastructure to be predetermined in a <em>BOSH release</em>, which is basically a blueprint of your entire infrastructure. This approach produces <strong>consistency</strong> amongst deployments, which in turn provides for a few key benefits:
+          </p>
+          <ul>
+            <li><strong>Cloud agnostic:</strong> BOSH guarantees consistency across deployments, but also does so across clouds. BOSH has Cloud Provider Interfaces (CPI's) for basically any major IaaS out there, including AWS, VMWare and OpenStack. By using BOSH to manage your infrastructure, you're making your deployments portable across IaaS providers! 
+            <li><strong>Health monitoring and recoverability:</strong> BOSH knows exactly in which state everything should be. Therefore it can recover processes and VMS as soon as it notices they've stopped running or have become unresponsive.</li>
+            <li><strong>Friendly towards change:</strong> Since you've already predetermined every single aspect of your infrastructure, the effects future changes might have with regard to your infrastructure become much more clearly visible.</li>
+            <li><strong>Scalability by design:</strong> Scale is just another mode of change. You'll want your infrastructure to perform consistently at any scale. BOSH makes it natural for you to think about scale and any other mode of change in a way that ensures consistency.</li>
+            <li><strong>Instant DevOps:</strong>BOSH has a holistic approach towards systems, bringing development and operations closer together. The demand for consistency requires operators and developers to work closer together to be able to understand properly how changes to an application are going to affect the bigger picture and vice versa.</li>
+          </ul>
+
+          <p>
+            The above should give a basic idea of what makes BOSH such a powerful tool. Also, it should be clear now that using BOSH will affect the way an operator goes about his operations. At Dozer, we believe that those using BOSH will always end up with better designed, more redundant and healthier systems. More info and documentation about BOSH can be found <a>here</a> (More links down below). The easiest way to get started is by using <a>BOSH Lite</a>. Once you're ready for production, check back here and we'll get you started in a minute!
```

Comment:
Dozer should be the easiest way to get started. Maybe better to not mention bosh-lite on our frontpage.